src/investarena/components/TradingForm/TradingForm.js

```diff
@@ -89,7 +110,7 @@ const TradingForm = ({
 
       <div className="st-trading-form__button-wrap">
         <TradeButton size="large" type={side} onClick={handleTradeButtonClick}>
-          {`${amount} ${baseCurrency} ${side}`}
+          {selectLocale()}
```

Comment:
Don't pass `amount` and `baseCurrency` to `FormattedMessage` - translate only `side` (Buy/Sell)